File: calculadora.java
import java.util.Scanner;

public class calculadora {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        System.out.print("Digite o primeiro número: ");
        int primeiro_numero = scanner.nextInt();
        
        System.out.print("Digite o segundo número: ");
        int segundo_numero = scanner.nextInt();
        
        System.out.println("Resultado da adição: " + (primeiro_numero + segundo_numero));
        
        System.out.println("Resultado da subtração: " + (primeiro_numero - segundo_numero));
        
        System.out.println("Resultado da multiplicação: " + (primeiro_numero * segundo_numero));
        
        if (segundo_numero != 0) {
            System.out.println("Resultado da divisão: " + ((double) primeiro_numero / segundo_numero));
        } else {
            System.out.println("A divisão por zero não é permitida.");
        }
        
        scanner.close();
    }
}

File: calculadora.c
#include <stdio.h>

int main() {
    
    int num1, num2;
    
    printf("Digite o primeiro numero: ");
    scanf("%d", &num1);
    printf("Digite o segundo numero: ");
    scanf("%d", &num2);
    
    printf("Resultado da adicao: %d + %d = %d\n", num1, num2, num1 + num2);
    
    printf("Resultado da subtracao: %d - %d = %d\n", num1, num2, num1 - num2);
    
    printf("Resultado da multiplicacao: %d * %d = %d\n", num1, num2, num1 * num2);
    
    if(num2 != 0) {
        printf("Resultado da divisao: %d / %d = %.2f\n", num1, num2, (double)num1 / num2);
    } else {
        printf("Divisao por zero nao eh permitida.\n");
    }
    
    return 0;
}